scripts/extract_validation_messages.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import yaml, os, shutil
+
+dir = os.path.dirname(__file__)
```

Comment:
Not tremendously important, but `dir` is a builtin in python and it's best to avoid reassigning builtin names to something else. (The most common culprit is `set`)